src/test/regress/sql/sequences_create.sql

```diff
@@ -0,0 +1,17 @@
+CREATE SCHEMA sequences_schema;
+SET search_path TO sequences_schema;
+
+CREATE SEQUENCE seq_0;
+ALTER SEQUENCE seq_0 AS smallint;
+
+CREATE TABLE seq_test_0 (x bigint, y bigint);
```

Comment:
could we expand the test slightly more? Say, we can have multiple sequences on a single table and also use bigserial columns.

Also, transactions are always tricky with Citus. Can you say do the same thing on a tx block as well?

```SQL
BEGIN;
INSERT INTO seq_test_0 VALUES (1,2);
select max(z)<nextval('renamed_seq') as check_sanity from seq_test_0 ;
COMMIT;
```

Or, use some different syntax from create sequence like:
```SQL
CREATE SEQUENCE bigint_sequence_3 INCREMENT 10;
```
And, make sure that Citus properly does increment?

Also, INSERT .. SELECTs are interesting for Citus' perspective. Would be interesting to have tests involving `INSERT .. SELECT`